resources/views/includes/slider/scripts.blade.php

```diff
@@ -31,6 +31,12 @@ function disableTabIndexOfInvisibleElements (sliderWrapper, slides) {
     }
 
     new Swiper('#swiper-{{ $id }}', {
+        @if($keyboardScroll)
+        keyboard: {
+            enabled: true,
+            onlyInViewport: true,
```

Comment:
If you want, you can omit `onlyInViewport` because is `true` by default.
https://swiperjs.com/swiper-api#keyboard-control